processes/fulfillURLManifest.py

```diff
@@ -62,31 +62,37 @@ def update_manifest(self, metadataObject, bucketName, currKey):
 
         metadataJSON = json.loads(metadataObject['Body'].read().decode("utf-8"))
         metadataJSONCopy = copy.deepcopy(metadataJSON)
+
+        #Checking if record asscociated with manifest is public domain or in copyright
+        checkRightsStatus = 'in copyright'
+        checkRightsStatus = self.checkMetaData(metadataJSON, checkRightsStatus)
+
+        if checkRightsStatus == 'in copyright':
```

Comment:
instead of indenting all of this code under this if statement, we could do

```
if rights_status == 'in copyright':
  return

Reply:
Sure